pandas/tests/resample/test_datetime_index.py

```diff
@@ -1039,8 +1039,12 @@ def test_resample_segfault(unit):
         all_wins_and_wagers, columns=("ID", "timestamp", "A", "B")
     ).set_index("timestamp")
     df.index = df.index.as_unit(unit)
-    result = df.groupby("ID").resample("5min").sum()
-    expected = df.groupby("ID").apply(lambda x: x.resample("5min").sum())
+    msg = "DataFrameGroupBy.resample operated on the grouping columns"
+    with tm.assert_produces_warning(FutureWarning, match=msg):
+        result = df.groupby("ID").resample("5min").sum()
```

Comment:
`resample` gives a warning? can we avoid that? (Does it operate on the grouping columns?)

Reply:
Then it should be deprecated and mentioned in the whatsnew also, right?

Reply:
Ah, yes, I missed mentioning it there. Added.